src/NHSD.BuyingCatalogue.Identity.Api/Startup.cs

```diff
@@ -59,15 +61,14 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddControllers();
             services.AddControllersWithViews();
-            services.AddAuthentication();
         }
 
         public void Configure(IApplicationBuilder app)
         {
             app.UseSerilogRequestLogging(opts =>
             {
-                opts.EnrichDiagnosticContext = LogHelper.EnrichFromRequest;
-                opts.GetLevel = LogHelper.ExcludeHealthChecks;
+                opts.EnrichDiagnosticContext = Infrastructure.LogHelper.EnrichFromRequest;
+                opts.GetLevel = Infrastructure.LogHelper.ExcludeHealthChecks;
```

Comment:
Why remove the `using` alias?

I think renaming/refactoring `LogHelper` would probably be a better long-term option (the name is vague and `ExcludeHealthChecks` is also badly named in my opinion – probably a tech debt ticket though.

Reply:
I find that style of using confusing (one to add to the next coding style meeting). The name was clashing so had to be explicit.

I agree the name of the LogHelper & ExcludeHealthChecks should be changed.